cloud_functions/token_refresh/streams/strava.py

```
@@ -0,0 +1,79 @@
+        raise ValueError("Strava credentials not found in Secret Manager")
+
+    token_response = client.refresh_access_token(
+        client_id=client_id,
+        client_secret=client_secret,
+        refresh_token=decrypt_token(refresh_token, encryption_key),
+    )
+
+    if time.time() > token_response["expires_at"] - 3600:  # Create a buffer for 1 hour
+        new_tokens = {
+            "accessToken": encrypt_token(
+                token_response["access_token"], encryption_key
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)